.tool-versions

```diff
@@ -1 +1 @@
-ruby 2.7.1
```

Comment:
we probably wanna stay on 2.7.1 here.

Reply:
Can we pair on this later? I keep running into this issue.

Reply:
sure! ill have some time around 4:30 today if that works for you